TestStack.FluentMvcTesting/ControllerResultTest/ShouldRedirectTo.cs

```diff
@@ -117,7 +117,10 @@ public RouteValueDictionary ShouldRedirectTo(MethodInfo method, RouteValueDictio
             var redirectResult = (RedirectToRouteResult)ActionResult;
 
             if (redirectResult.RouteValues["Controller"] == null)
```

Comment:
Should this be `redirectResult.RouteValues["Controller"] == null && typeof(TController) == typeof(T)`?

Otherwise, if the action returned a redirect to the same controller, but you were checking for a different one then it wouldn't throw when it should.

You can probably add a test case for that if there isn't one already.